spec/system/client_balance/admin_consults_client_balance_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+
+describe 'Administrador consulta o saldo de um cliente' do
+  it 'a partir da tela inicial' do
+    admin = create(:admin, :approved)
+
```

Comment:
Seria bom testar se o caminho atual é igual ao que esta sendo testado em baixo
```suggestion
expect(current_path).to eq(client_balances_path)
```

Reply:
Feito!